apps/dapp/src/components/Pages/Ascend/components/LBPForm/index.tsx

```diff
@@ -1,114 +1,81 @@
 import { useState } from 'react';
-import { BigNumber } from 'ethers';
 import styled, { css } from 'styled-components';
-
 import { Pool } from 'components/Layouts/Ascend/types';
 import env from 'constants/env';
 import { Button } from 'components/Button/Button';
+import { usePoolContract } from './use-pool-contract';
+import { useFactoryContract } from './use-pool-factory';
+import { format, parse } from 'date-fns';
+import { formatUnits } from 'ethers/lib/utils';
+import { Link } from 'react-router-dom';
+import { DecimalBigNumber } from 'utils/DecimalBigNumber';
+import { ZERO } from 'utils/bigNumber';
 
 interface Values {
   id?: string;
   name: string;
-  overview: string;
+  symbol: string;
   releasedToken: string;
   accruedToken: string;
-  startWeight: [BigNumber, BigNumber];
+  startWeight1: DecimalBigNumber;
+  startWeight2: DecimalBigNumber;
   startDate: Date;
   endDate: Date;
   fees: number;
   address: string;
+  endWeight1: DecimalBigNumber;
+  endWeight2: DecimalBigNumber;
 }
 
 const getInitialValues = (pool?: Pool): Values => {
   if (!pool) {
     return {
-      overview: '',
+      symbol: '',
       name: '',
       releasedToken: env.tokens.temple.address,
       accruedToken: env.tokens.frax.address,
-      startWeight: [BigNumber.from(0), BigNumber.from(0)],
+      startWeight1: DecimalBigNumber.fromBN(ZERO, 18),
+      startWeight2: DecimalBigNumber.fromBN(ZERO, 18),
       startDate: new Date(0),
       endDate: new Date(0),
       fees: 1,
       address: '',
+      endWeight1: DecimalBigNumber.fromBN(ZERO, 18),
```

Comment:
for these ZERO weights, there is a `DBN_ZERO` constant exported from the DecimalBigNumber file.  should probably just use the constant for all these 0 values?

Reply:
Agreed 👍

Reply:
https://github.com/TempleDAO/temple/pull/461/commits/bb507608688ba733e68f2319522ec1215bc9784f